vividus-plugin-json/src/main/java/org/vividus/json/steps/JsonSteps.java

```diff
@@ -485,6 +490,25 @@ public void assertNumberOfJsonElements(String json, String jsonPath, ComparisonR
         assertJsonElementsNumber(jsonPath, actualNumber, comparisonRule, elementsNumber);
     }
 
+    /**
+     * Validates json against json schema
+     * @param json JSON
+     * @param schema JSON schema
+     */
+    @Then("JSON `$json` is valid against schema `$schema`")
+    public void validateJsonAgainstSchema(String json, String schema)
+    {
+        JsonSchemaFactory factory = JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V202012);
```

Comment:
I guess, it makes sense to version of the schema specified in it, otherwise we should default to 2020-12

Reply:
@valfirst  could you please elaborate

Reply:
@ikalinin1 the schema itself specfies its version "$schema": "https://json-schema.org/draft/2020-12/schema",